content/influxdb/cloud/account-management/billing.md

```diff
@@ -98,9 +98,9 @@ On the **Billing page**, view the total limits available for the Free Plan.
 
 ## Review and resolve plan limit overages
 
-If you exceed your plan's [limits](/influxdb/cloud/account-management/pricing-plans), you'll receive a notification in the {{< cloud-name "short" >}} user interface (UI) **Usage** page.
+If you exceed your plan's [adjustable quotas or limits](/influxdb/cloud/account-management/limits/), you'll receive a notification in the {{< cloud-name "short" >}} user interface (UI) **Usage** page.
 
-If exceed the series cardinality limit, InfluxDB adds a rate limit event to your **Usage** page for review, and begins to reject write requests. To start processing write requests again, do the following as needed:
+If you exceed the series cardinality limit, InfluxDB adds a rate limit event to your **Usage** page for review, and begins to reject write requests. To start processing write requests again, do the following as needed:
```

Comment:
we only reject write requests which include new series?  i.e. you are not allowed to create any new series... but if you have writes which do not create new series.... I believe these are accepted.